src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/sentiment/WordListSentimentDetector.scala

```diff
@@ -0,0 +1,77 @@
+package com.microsoft.partnercatalyst.fortis.spark.transforms.sentiment
+
+import java.io.{File, IOError, IOException}
+import java.util.concurrent.ConcurrentHashMap
+
+import com.microsoft.partnercatalyst.fortis.spark.logging.Loggable
+import com.microsoft.partnercatalyst.fortis.spark.transforms.ZipModelsProvider
+import com.microsoft.partnercatalyst.fortis.spark.transforms.nlp.Tokenizer.tokenize
+import com.microsoft.partnercatalyst.fortis.spark.transforms.sentiment.SentimentDetector.{NEGATIVE, NEUTRAL, POSITIVE}
+
+import scala.io.Source
+
+@SerialVersionUID(100L)
+class WordListSentimentDetector(
```

Comment:
There could be an interface for it, but right now there's only two implementations which are both wrapped by a single class so there wouldn't be much value in it.